### DISCUSSION TITLE
Developer pack Evidence issue

### DISCUSSION THREAD

**Discussion by Rehankhanpy** [community/community#143403] [GitHub Education]
### Hi everyone,

Hello I can't upload a photo of evidence for Student developer pack because only front camera is opening please fix it, and I can't upload anything

**Comment by Gvolexe** [comment]
Hi there,

Unfortunately, GitHub's Student Developer Pack verification currently only supports capturing photos directly from your device's camera, and it doesn’t provide an option to upload an existing image file. If your camera defaults to the front-facing one and doesn’t allow switching, try using a different browser or device (such as a desktop computer) to see if it improves the experience.

If you're on a mobile device and want to use the back camera, try using a mobile browser that may better support back camera access, such as Chrome, Firefox, or Safari. Each browser may handle camera permissions differently and allow switching between cameras.

If you’re still facing issues, I recommend contacting GitHub Education support. They may be able to offer a workaround or help resolve the camera restriction.

Hope this helps, and good luck with your application!
